site/scripts/docs.js

```diff
@@ -25,7 +25,7 @@ const config = {
         md += formatFlags(info.flags)
         md += commandListSubCommandDisplay(info.commands)
         md += commandExamples(info.examples)
-        if (info.commands.length) {
+        if (info.commands.length !== 0) {
```

Comment:
Many changes in this PR follow this pattern: this enforces `length` comparison always use the same pattern (`=== 0` or `!== 0`), for consistency.